data/volunteers.js

```diff
@@ -9,11 +9,36 @@ export default [
     title: 'Front-End Developer',
     image: 'simi-damani.jpg',
   },
+  {
+    name: 'Nick Holden',
+    title: 'Software Engineer',
+    image: 'nick-holden.jpg',
+  },
+  {
+    name: 'Dashiel Lopez Mendez',
+    title: 'Software Engineer',
```

Comment:
There was no title in the image name in the Google Drive, so I Googled Dash and this seemed like an appropriate title. Would you prefer something else, @laconc?

Reply:
previously, we used "Infrastructure" for @laconc -- whichever he prefers works for me!